File: app/src/main/java/com/hoversoftsoln/esta_fort/home/data/Service.java
package com.hoversoftsoln.esta_fort.home.data;

import com.hoversoftsoln.esta_fort.core.data.BaseDao;

import java.util.Map;

public class Service extends BaseDao<Service> {

    private String id, name, desc, image_url;
    private long issue_time;

    @Override
    public Service maptoData(Map<String, Object> _datamap) {
        return null;
    }

    @Override
    public Map<String, Object> datatoMap(Service _data) {
        return null;
    }

    public String getId() {
        return id;
    }

    private void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    private void setName(String name) {
        this.name = name;
    }

    public String getDesc() {
        return desc;
    }

    private void setDesc(String desc) {
        this.desc = desc;
    }

    public String getImage_url() {
        return image_url;
    }

    private void setImage_url(String image_url) {
        this.image_url = image_url;
    }

    public long getIssue_time() {
        return issue_time;
    }

    private void setIssue_time(long issue_time) {
        this.issue_time = issue_time;
    }
}

File: app/src/main/java/com/hoversoftsoln/esta_fort/core/data/Service.kt
package com.hoversoftsoln.esta_fort.core.data

import com.google.firebase.firestore.PropertyName

data class Service(
        @get:PropertyName("title")
        @set:PropertyName("title") var title: String = "",
        @get:PropertyName("desc")
        @set:PropertyName("desc") var desc: String = "",
        @get:PropertyName("image")
        @set:PropertyName("image") var image: String = "",
        @set:PropertyName("issue_date")
        @get:PropertyName("issue_date") var issue_date: Long = 0
)